js/data.js

```diff
@@ -1,8 +1,19 @@
 /* exported data */
-
 var data = {
   view: 'entry-form',
   entries: [],
   editing: null,
   nextEntryId: 1
 };
+
+var previousData = localStorage.getItem('code-journal-entries');
+
+if (previousData !== null) {
+  data.entries = JSON.parse(previousData);
+}
+
+function windowBeforeUnload(event) {
+  localStorage.setItem('code-journal-entries', JSON.stringify(data.entries));
```

Comment:
You shouldn't be only saving `data.entries` into localStorage, you want to save the entire `data` object so you can preserve the next entry id, and the other properties to be used in later features.